docs/rules/0148/uid-format.md

```diff
@@ -16,8 +16,9 @@ field, as mandated in [AIP-148][].
 ## Details
 
 This rule looks on for fields named `uid` and complains if it does not have the
-`(google.api.field_info).format = UUID4` annotation or has a format other than
-`UUID4`.
+`(google.api.field_info).format = UUID4` annotation or has a
+[format](https://github.com/googleapis/googleapis/blob/master/google/api/field_info.proto#L54)
```

Comment:
Would you mind doing a few things:
1. Put this in a named link down at the bottom e.g. `[format]: https://....` and changing this to `[format][]`
2. Remove the Line number - when targeting `master` it is potentially inaccurate

Alternatively, just link to AIP-202 where all of the formats are described.